lib/nunjucks/globals.js

```diff
@@ -27,6 +27,80 @@ exports.getNunjucksCode = function (path) {
   return content.replace(/{%\s*extends\s*\S*\s*%}\s+/, '')
 }
 
+/**
+ * This helper function takes a path of a *.erb file and
+ * returns the Rails syntax inside that file without markdown data
+ *
+ * @param {string} path - Path to ERb file
+ * @returns {string} ERb code for the file
+ */
+exports.getRubyOnRailsCode = (path) => {
+  const { content } = matter(readFileSync(path, 'utf-8'))
+
+  return content
+}
+
+/**
+ * This helper function takes an component name and returns
+ * a URL for the full documentation for that Rails helper.
+ *
+ * @param {string} componentName - Name of component
+ * @returns {string} URL - Link to documentation page
+ */
+exports.getRailsLink = (componentName) => {
```

Comment:
@peteryates @paulrobertlloyd this seemed like the simplest way to work out which page to link to - but open to other ideas!

Reply:
Depends how far you want to go with this, but you could create 2 arrays, one for components, and one for form elements, then cycle through them:

```js
const links = {}

const components = ['accordian', 'breadcrumb']
const formElements = ['character-count', 'date-input']
  
for (const component in components) {
  links[component] = `https://govuk-components.netlify.app/components/${component}/`
}

for (const formElement in formElements) {
  links[formElement] = `https://govuk-form-builder.netlify.app/form-elements/${formElement}/`
}
```

(Top of my head coding, may need tweaking). This way, you only need to update each array and 2 URL paths should any of those aspects change.

Reply:
It's slightly more complicated as some of the components have the prefix `/building-blocks/` or `/helpers/` or `/introduction/`, plus there's a few mismatches (`fieldset` vs `fieldsets`) - but something like this could work!

I was initially going to just link to either of the 2 homepages – but I think direct linking to the relevant pages is valuable?

Not sure where we'd link for any Rails examples for the patterns though... Some of the styles (eg Links) need doing too.